xbmc/pvr/PVRManager.h

```diff
@@ -70,6 +71,21 @@ namespace PVR
     ManagerStateStarted
   };
 
+  struct PVRManagerEvent
+  {
+    virtual ~PVRManagerEvent() {};
+  };
+
+  namespace PVRManagerEvents
+  {
+    struct Starting    : PVRManagerEvent {};
```

Comment:
@FernetMenta ask @tamland. he wanted me to do it like he does with addons (at least this is what i understood from his review).